src/openms/source/ANALYSIS/XLMS/OPXLHelper.cpp

```diff
@@ -195,16 +193,9 @@ namespace OpenMS
     if (low_it != up_it) // if they are not equal, there are matching precursors in the data
     {
       // found_matching_precursors = true;
-
-// don't access this vector from two processing threads at the same time
-#ifdef _OPENMP
-#pragma omp critical
```

Comment:
Why not needed anymore

Reply:
related to the removed nested parallel loop. Since the function itself is not parallel anymore and different calls to this function don't access the same instances of these vectors, there is no need for a critical section